app/models/measure.rb

```diff
@@ -101,6 +101,10 @@ class Measure < Sequel::Model
 
   delegate :third_country?, :excise?, :vat?, :trade_remedy?, to: :measure_type, allow_nil: true
 
+  def universal_waiver_applies?
+    measure_conditions&.any?(&:universal_waiver_applies?)
```

Comment:
Whats the need for the safe navigation operator? `measure_conditions` is a relationship isn't it? So should not ever be nil?

Reply:
No need. Have removed